salang/app/login.tsx

```
@@ -106,6 +111,8 @@ export default function LoginScreen() {
 
       {/* 카카오 버튼 */}
       <TouchableOpacity
+        onPress={handleKakaoLogin}
+        className='absolute left-[35px] top-[520px] h-[70px] w-[340px] items-center justify-center rounded-xl bg-white px-3'
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)